specs/ignore3vl.toml

```diff
@@ -0,0 +1,27 @@
+[setup]
+statements = [
+    "create table t(a array(integer))",
+    "copy t from 's3://crate.benchmark.data/table_int_array.json.gz' with (compression = 'gzip')",
+    "refresh table t"
+]
+
+# All 250k rows match
+#[[queries]]
+#statement = "select count(*) from t where not 101 = any(a)"
```

Comment:
Can we remove these?

Reply:
Without comments it's hard to infer what they're for - looks like dead code. I'd remove them unless you'd like to chart the results of these queries as well, in which case I'd fully include them